collectors/python.d.plugin/adaptec_raid/README.md

```diff
@@ -6,53 +6,65 @@ sidebar_label: "Adaptec RAID"
 
 # Adaptec RAID controller monitoring with Netdata
 
-Collects logical and physical devices metrics.
+Collects logical and physical devices metrics using `arcconf` command-line utility.
+
+Executed commands:
+
+- `sudo -n arcconf GETCONFIG 1 LD`
+- `sudo -n arcconf GETCONFIG 1 PD`
 
 ## Requirements
 
-The module uses `arcconf`, which can only be executed by root.  It uses
-`sudo` and assumes that it is configured such that the `netdata` user can
-execute `arcconf` as root without password.
+The module uses `arcconf`, which can only be executed by `root`. It uses
+`sudo` and assumes that it is configured such that the `netdata` user can execute `arcconf` as root without a password.
 
-Add to `sudoers`:
+- add to the `sudoers`
```

Comment:
Why do we not specify the whole path `/etc/sudoers`?

Reply:
It may not always be in that location. It’s _very_ rare for this to be the case these days, but still possible.

Reply:
Thanks for the explanation!